src/human.js

```diff
@@ -69,8 +69,8 @@ module.exports = class extends EventEmitter {
     if (pack.length === 1)
       return this.pick(0)
 
-    if (this.useTimer)
-      this.time = 20 + 5 * pack.length
+    if (this.useTimer > 0)
+      this.time = parseInt(this.useTimer) + parseInt(pack.length)
```

Comment:
It's probably idiomatic to use `parseInt(foo, 10)` as per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt.